README.md

```diff
@@ -1,2 +1,56 @@
-# ktor-cli
-Command-line tool for creating Ktor projects
+# Ktor CLI
+Ktor CLI is a command-line tool that brings the capability to create, run, and interact with your [Ktor](https://ktor.io/) application from the command line.
+
+## Install CLI tools
+
+### macOS
+
+You can install Ktor CLI on macOS using [Homebrew](https://brew.sh/) as follows:
+1. Add a Ktor repository using the `brew tap` command:
+   ```
+   brew tap ktorio/ktor\
+   ```
+2. Install Ktor CLI using `brew install`:
+   ```
+   brew install --build-from-source ktor\
+   ```
+
+### Linux
+
+On Linux, you can install Ktor CLI using [snaps](https://snapcraft.io/):
```

Comment:
The utility is called `snap`. `Snaps` are packages. 
`Ktor CLI` is `Ktor command line interface` expanded, but you only can install a program or application that communicates via command line interface.